src/Console/Application.php

```diff
@@ -121,7 +121,7 @@ public function getSymfonyStyle(): SymfonyStyle
         return $this->symfonyStyle ?? throw new \LogicException('SymfonyStyle not available yet.');
     }
 
-    public function getCommand(): Command
+    public function getTask(): Command
     {
         return $this->command ?? throw new \LogicException('Command not available yet.');
     }
```

Comment:
I'm uncomfortable with this change.
If we go this way, the exception message must be updated too!

But according to the signature, I prefer to keep `getCommand`. And so, I prefer to revert changes in src/Attribute/AsOption.php and src/Attribute/AsTaskArgument.php